src/message.js

```diff
@@ -762,7 +762,7 @@ async function createVerificationObject(signature, literalDataList, keys, date =
       )) {
         throw new Error('Signature is expired');
       }
-      return verified;
+      return !!sig.verified;
```

Comment:
I think you can just
```suggestion
      return true;
```
here, rather than relying on the caching property

Reply:
But that should really be true at this point, I wanted to rely on it to catch any unexpected behaviour

Reply:
Yeah, but.. we rely on `verify` to throw in all other places, I think it's a bit confusing not to do the same here